app/main.py

```diff
@@ -2,0 +1,17 @@
+def copy_file(command: str) -> None:
+    parts = command.split()
+    if len(parts) != 3 or parts[0] != "cp":
+        return
+
+    source_file = parts[1]
+    destination_file = parts[2]
+
+    if source_file == destination_file:
```

Comment:
The check for the source and destination being the same is correct. Consider adding a comment to explain why this check is necessary.